views/layouts/application.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+
+<html>
```

Comment:
Don't forget to add a class layout:

```suggestion
<html class="layout-default">
```

This can be added later but you will need that for styling layouts